.cargo/config.toml

```diff
@@ -0,0 +1,7 @@
+[alias]
+pyo3_doc = "doc --lib --no-default-features --features=full --no-deps --workspace --open --exclude pyo3-macros --exclude pyo3-macros-backend"
+pyo3_doc_scrape = "doc --lib --no-default-features --features=full --no-deps --workspace --open --exclude pyo3-macros --exclude pyo3-macros-backend -Z unstable-options -Z rustdoc-scrape-examples=examples"
+pyo3_doc_internal = "doc --lib --no-default-features --features=full --no-deps --workspace --open --document-private-items -Z unstable-options -Z rustdoc-scrape-examples=examples"
+
+[build]
+rustdocflags = ["--cfg", "docsrs"]
```

Comment:
I suspect adding this flag here means that `cargo doc` will no longer work on stable rust? That seems undesirable.

Reply:
Unfortunately there seems to be no way to pass these over the command line when using `cargo doc`. It has to be an environment variable. I used `cargo doc` because you can't pass the `--workspace` to rustdoc. I'd be happy to be wrong here :)

Reply:
Ah ok, let's leave it like this for now and then maybe I'll have a play with it another time if it bothers me 😅